examples/analog_hamiltonian_simulation/01_Introduction_to_Aquila.ipynb

```diff
@@ -7,175 +7,205 @@
     "tags": []
```

Comment:
<p>1) "greateer" -&gt; "greater" (at several places)</p><p>2) "4. All the fields in the driving field have to end at the same time point." -&gt; "4. All field have to have the same duration"</p><p>3) We can drop 5, as it is guaranteed by using the BDK.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>